activity_hub.php

```diff
@@ -244,11 +255,30 @@ function progress_snapshot_table($show_filtered_projects, $show_filtering_links,
             continue;
         }
 
-        $desired_states = [];
+        echo "\n<tr>";
```

Comment:
After thinking about this some, we need to put this in an `if ($stage->id == "SR")` block to prevent breaking other sites that might have a stage that isn't SR. That way we retain the old functionality for non-SR stages and we use this new one for SR. Your updates to `summarize_stage()` already handle this case, we just need to update this spot.

Reply:
Good point, and done.